src/main/java/org/cloudfoundry/promregator/cfaccessor/CFWatchdog.java

```diff
@@ -36,7 +36,7 @@
 	
 	@Scheduled(fixedRateString = "${cf.watchdog.rate:60}000", initialDelayString = "${cf.watchdog.initialDelay:60}000")
 	@SuppressWarnings("unused")
-	private void connectionWatchdog() {
+	protected void connectionWatchdog() {
```

Comment:
Upon review of our own code, I discovered we were doing something unexpected that caused visibility issues here at startup. I have since fixed our side and pushed the change back to _private_ visibility.